content/en/examples/service/load-balancer-example.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: load-balancer-example
```

Comment:
@medmedchiheb - if you want, you could use shared labels prefixed app.kubernetes.io, as per [recommended labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels)

(either instead of or as well as the plain `app` label).

Reply:
yeah